docs/templates/PACKAGE_MAIN_DOCUMENTATION.MD

```diff
@@ -0,0 +1,105 @@
+**📦 Resources**
+
+<!-- RESOURCES:
+* required
+
+Listing of resources related to the topic including running versions of used code snippets or other applications. 
+Resource links are prefixed with the "🔗" icon
+-->
+
+Example:
+> [🔗 GitHub](https://github.com/BioPhoton/rx-angular-cdk-coercing).
+
+
+# ⛽ Motivation
+
+<!-- MOTIVATION:
+* required
+
+Motivation is here to point out the value for the consumer. If possible we can make clear in which situations the impact will get visible most.
+
+Importent to cover in this section is:
+- Theoretical background to understand the problem
+- The problem and pain points in detail
+- Sketch a optimal solution for the problem
```

Comment:
```suggestion
This section should cover at least these main points:
- Problem description. What are existing pain points?
- Theoretical background to understand the problem
- Description of different solution approaches
- Sketch the optimal solution for the problem and explain why (should be our solution then :1st_place_medal:)
```

Reply:
Oh I have just seen there is a dedicated chapter for different approaches. Do we really need to dedicate a whole section to it?